website/versioned_docs/version-19.2.0/index.md

```diff
@@ -221,7 +219,7 @@ See our
 ## Resources
 
 - [API Documentation](https://pptr.dev/api)
-- [Guides](https://pptr.dev/guides)
+- [Guides](https://pptr.dev/category/guides)
```

Comment:
can we remove the `category` from the URL like it was? /guides looks much better than category/guides

Reply:
AFAIS, no. We can add a redirect though. Let me do that.